src/api/Volkszaehler.cpp

```diff
@@ -304,22 +305,19 @@ int vz::api::curl_custom_debug_callback(CURL *curl, curl_infotype type, char *da
 	case CURLINFO_END:
 		if (end)
 			*end = '\0'; // terminate without \n
```

Comment:
please remove this as well

Reply:
or remove the \n in the buffer but not in the data:
if (end !== std::string::npos) { buffer[end] = 0;}